### PR TITLE
Support more flexible, faster development cycles

### PR DESCRIPTION
**Pull request by botimer**
There are two small commits here with detailed messages. They add "-repeat watch" support and allow configuring the replica path. This combination enhances the ease of use and performance for development.

Here is a handy shell alias to start syncing from the current directory to an active Docker machine instance. This does require unison-fsmonitor to be set up on the client as well.

```
function dsync() {
    MACHINE=""
    if [[ -n $1 ]]; then
        MACHINE=$1
    else
        MACHINE="$(dm active)"
    fi

    HOST="$(dm ip $MACHINE)"
    PORT=5000
    if [[ -n $2 ]]; then
        PORT=$2
    fi

    unison . socket://$HOST:$PORT/ -ignore 'Path .git' -ignore 'Path tmp' -auto -batch -prefer . -repeat watch -fastcheck true
}
```

### CODE REVIEW

File: unison-link.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+ln -s /bin/unison-fsmonitor-$UNISON_VERSION /bin/unison-fsmonitor
```
Comment (leighmcculloch) [inline unison-link.sh:3]: Let's move this into the `unison-install.sh` script, since this shouldn't need to be linked on every run.

Comment (leighmcculloch) [inline unison-link.sh:3]: Oh right, it needs to happen each time, because the ENV `UNISON_VERSION` could be changed between runs. :smile:

**Comment by leighmcculloch** [comment]
@botimer Thanks so much for submitting this PR! It looks good. Before I merge it, can you remove the second commit since that feature was already provided by #6, and update the commit message of the first commit to remove mention of `-repeat` which is a detail. e.g. `Install and link unison-fsmonitor.` Once that's done, mention me here and I'll merge!

**Comment by botimer** [comment]
@leighmcculloch This is now cleaned up as requested. Thanks for checking it out!

File: Dockerfile [hunk 2]
```diff
@@ -26,6 +26,10 @@ RUN apt-get update -y \
 ENV UNISON_VERSION=2.48.3
 ENV UNISON_WORKING_DIR=/unison
 
+# Link the unison-fsmonitor for the right version to support -repeat watch
```
Comment (leighmcculloch) [inline Dockerfile]: Drop this comment, I think it's unnecessary, otherwise it's a go!

Comment (botimer) [inline Dockerfile]: Done!

**Pull request merged**